linalg/sundials.hpp

```diff
@@ -586,10 +586,10 @@ class CVODESSolver : public CVODESolver
 
        @note On input, the values of t and dt are used to compute desired
        output time for the integration, tout = t + dt. */
-   virtual void Step(Vector &x, double &t, double &dt);
+   void Step(Vector &x, double &t, double &dt) override;
 
    /// Solve one adjoint time step
-   virtual void StepB(Vector &w, double &t, double &dt);
+   void StepB(Vector &w, double &t, double &dt) override;
```

Comment:
Thanks for finding this @cjvogl. I committed your suggestion. I had not tried building with sundials. There could be other cases like this that haven't been found, but the main goal of the PR is to fix most cases so that the correct usage of `virtual` and `override` is followed by example.